nncf/experimental/torch/nas/bootstrapNAS/search/search.py

```diff
@@ -63,15 +64,17 @@
 
 class EvolutionaryAlgorithms(Enum):
     NSGA2 = 'NSGA2'
-
+    RNSGA2 = 'RNSGA2'
 
 class SearchParams:
     """
     Storage class for search parameters.
     """
     def __init__(self, num_evals: float, num_constraints: float, population: float,
                  seed: float, crossover_prob: float, crossover_eta: float,
-                 mutation_prob: float, mutation_eta: float, acc_delta: float, ref_acc: float):
+                 mutation_prob: float, mutation_eta: float, acc_delta: float, ref_acc: float,
+                 aspiration_points: np.ndarray, epsilon: float, weights: np.ndarray,
+                 extreme_points_as_ref_points: bool):
```

Comment:
Seems like there is issue in the design of SearchParams class. 
There's a common structure that have parameters for both algorithms. But some of them are not needed for the NSGA. I'd  define 2 different classes.

Otherwise, support of the new algorithm would lead to a potential conflict with common structure.

If addition of the new algorithm leads to addition of parameters to the common structure, that are needed for a new algo, but not needed for an old one. I'd separate them

Reply:
Thanks, @ljaljushkin. We modify the code based on your suggestion.